subjects/devops/concat-string/README.md

```diff
@@ -0,0 +1,29 @@
+## concat_string
```

Comment:
The tile here is `snake_case` while the folder is `kebab-case` - to avoid confusion I would keep the same

Reply:
Done. Thanks.